qemu/softmmu/virtual_tlb.c

```diff
@@ -0,0 +1,106 @@
+#include <stdint.h>
```

Comment:
Is this file from QEMU? If not, I tend to rename it to `unicorn.c`.

Reply:
`unicorn.c` is a bit generic, how about `unicorn_vtlb.c`?

Reply:
yes, that's also fine